### ISSUE TITLE
Unable to resolve dependencies from artifactory requiring authorization

### ISSUE THREAD

**Issue by lemoony** [jmfayard/refreshVersions#288]
**Describe the bug**

```
15:27:06: Executing task 'refreshVersions'...

<-- 401 Unauthorized https://custom-repo.com/.../maven-metadata.xml (231ms, 91-byte body)

> Task :refreshVersions FAILED
1 actionable task: 1 executed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':refreshVersions'.
> ModuleId(group=de.xx.yyy, name=zzz) not found. Searched the following repositories:
  - file:/Users/pse/.m2/repository/
  - https://repo.spring.io/milestone/
  - https://repo.maven.apache.org/maven2/
  - https://jcenter.bintray.com/
  - https://custom-repo.com/

```

build.gradle:

```
repositories {
    mavenLocal()
    maven { url = uri("https://repo.spring.io/milestone") }

    mavenCentral()
    jcenter()
    maven {

        val artifactory_username: String by project
        val artifactory_password: String by project

        url = uri("https://custom-repo.com")
        credentials {
            username = artifactory_username
            password = artifactory_password
        }
    }
}
```

repo.gradle:

```
project.ext {
    artifactoryConfiguration = {
        url "https://custom-repo-com"
        credentials {
            username artifactory_username // defined in gradle.properties
            password artifactory_password // defined in gradle.properties
        }
    }
}

```

**Additional context**
Additional context

Gradle Version: 6.5.1
Kotlin Version: 1.4.10
RefreshVersions Version: 0.9.7 

**When downgrading to version 0.9.5 the build succeeds.**

**Comment by DanySK** [comment]
I'm being hit by this one as well. In particular, it does not play very well with github packages, which (unfortunately) require authorization even for public repos.

**Comment by LouisCAD** [comment]
Should be resolved in 0.10.0, freshly released.
